monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+# Empezando a Ayudar a Monero
+
+**_“La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta.”_ - 21 de agosto, 2019**
+---
+
+La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta. Un hecho poco conocido y emocionante es que Monero tiene la tercera cuenta de colaboradores más alta en todas las criptomonedas. Adicionalmente, hay muchos grupos de trabajo y proyectos que han evolucionado de la mano de Monero y es relativamente fácil empezar a ayudar.
+
+Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y como PUEDEN involucrarse. Bienvenidos a Monero. Esperamos que se queden.
+
+[Desarrolladores](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#developers)
+[Equipos de trabajo](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+[Únete a la conversación e involucrate](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#join)
+[Otras formas de ayudar](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#otherways)
+[Monero 101](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#monero101)
+[Minar Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#mining)
+
+### _Desarrolladores_
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
+
+La principal manera de empezar, es familiarizarse con la base del código, el estándar del protocolo cryptonote e interactuar con otros desarrolladores. Todos los desarrolladores se comunican a través de IRC en #monero-dev o #monero en los servidores de freenode. Encuentra aquí recursos para que los desarrolladores puedan empezar:
+
+- Código Fuente - [github.com/monero-project](https://github.com/monero-project)
+- Asuntos(Issues) abiertos - [github.com/monero-project/monero/issues](https://github.com/monero-project/monero/issues)
+- El ecosistema de Monero - [github.com/monero-ecosystem](https://github.com/monero-ecosystem)
```

Comment:
**Asuntos (Issues)**, there should be a space, shouldn't be?

Another suggestion, **Ecosistema de Monero** would be better, to follow the other bullets translations: is not _**El** laboratorio de investigación de Monero_, and so on.

Reply:
You're totally right. Changed. Thanks